api/v1/controllers/RSVPController.js

```diff
@@ -8,7 +8,10 @@ const config = require('ctx').config();
 
 const router = require('express').Router();
 function _isAuthenticated(req) {
-  return req.auth && (req.user !== undefined);
+  if(!(req.auth && (req.user !== undefined))) {
+    return false;
+  }
+  return services.RegistrationService.findAttendeeByUser(req.user).then((attendee) => attendee != null && attendee.get('status') == 'ACCEPTED');
```

Comment:
Let's not use a magic string, there should be a constant in the AttendeeModel

Reply:
I would need to do Attendee.STATUSES[0] to get the 'ACCEPTED' string, so there's still a magic number.